torch/csrc/Module.cpp

```diff
@@ -1356,6 +1356,13 @@ Call this whenever a new thread is created in order to propagate values from
     c10::impl::_force_tls_local_dispatch_key_set(local_keyset);
   });
 
+  py_module.def("_meta_in_tls_dispatch_include", []() {
+    auto local_keyset = c10::impl::tls_local_dispatch_key_set();
+    c10::DispatchKeySet key_set({at::DispatchKey::Meta});
+    auto k = key_set.highestBackendKey();
+    return local_keyset.included_.has_backend(k);
```

Comment:
nit: you could just do `.has_backend(c10::BackendComponent::MetaBit)` instead of making the keyset and pulling the backend component out of it.

Reply:
will do in a follow up